build-tools/stylelint/__tests__/no-motion-outside-of-mixin.test.js

```diff
@@ -64,29 +64,29 @@ describe('No motion outside of mixin', () => {
         }
       `);
       expect(result.errored).toBe(true);
-      expect(result.results[0].warnings).toEqual([
```

Comment:
Updated deps brought in new props to this returned object but the returned object is different for each of the test permutations. Made these slightly more generic by not checking the line and column

Reply:
How about 

```js
expect(result.results[0].warnings).toEqual([
   expect.objectContaining({
      rule: '@cloudscape-design/no-motion-outside-of-mixin',
      severity: 'error',
      text: `Property "${property}" should be directly under 'with-motion' helper (@cloudscape-design/no-motion-outside-of-mixin)`
   })
]);
```

The benefit of this pattern is that it will print you all object difference, not only one field. Every failed `expect` aborts the test

Reply:
Changed 👍🏻